Makefile

```diff
@@ -1,4 +1,6 @@
 $stdout.sync = true
+export VETS_API_ID  := $(shell id -u)
```

Comment:
Clever! It's somewhat unfortunate that our build user is named "vets-api", because `VETS_API_ID` doesn't scream out to me that this is actually a _user_ id. Maybe rename to `VETS_API_USER_ID`? 🤷‍♂

Reply:
Good idea!  Will do.